server.js

```diff
@@ -51,12 +55,12 @@ app.delete('/account/close/:accountId', (req, res) => {
     })
 });
 
-//Error handle
+// Error handling
 app.use(function(error, req, res, next) {
     res.status(500).send('500: Internal Server Error');
 });
 
 app.listen(app.get('port'), function(){
     console.log( 'Express started on http://localhost:' +
         app.get('port'));
```

Comment:
That's totally correct but maybe for cases like that is better to use a constant. Again is totally alright and it depends on what you prefer :) 

```suggestion
        const PORT = app.set('port', process.env.PORT || 3000);
        
        app.listen(PORT, function(){
               console.log( `Express started on http://localhost: ${PORT}`);
```